tox.ini

```diff
@@ -114,6 +114,17 @@ deps =
     wheel
 commands = python scripts/release.py {posargs}
 
+[testenv:publish_gh_release_notes]
+description = publish GitHub release notes after deployment
```

Comment:
It also creates the GitHub release in the first place - which was not clear to me, so worthwile to reword maybe.